docs/source/user/utilities/accuracy/accuracy-pitfalls.ipynb

```diff
@@ -23,17 +24,24 @@
     "differentially private way -- we will call this value $\\phi_{dp}(D)$. An $\\alpha$-level accuracy guarantee $a$ promises\n",
```

Comment:
The line above this one, "is a function of alpha and either sigma or accuracy"? Meaning accuracy is a function of accuracy?